xarray/core/common.py

```diff
@@ -1394,28 +1395,48 @@ def __getitem__(self, value):
 
 @overload
 def full_like(
-    other: DataArray, fill_value: Any, dtype: DTypeLikeSave = None
+    other: DataArray,
+    fill_value: Any,
+    dtype: DTypeLikeSave | None = None,
+    chunks={},
```

Comment:
```suggestion
    *,
    chunks={},
```
keyword only?

Also a little sketchy to use mutable dict as default value.

Reply:
Does that make it safer? I was copying the defaults we already have, see https://github.com/pydata/xarray/blob/087ebbb78668bdf5d2d41c3b2553e3f29ce75be1/xarray/core/dataset.py#L2203